app/views/pages/landing.html.erb

```diff
@@ -5,9 +5,13 @@
     <p class="govuk-body">
       <%= t(".blurb") %>
     </p>
-
-    <%= govuk_start_button(text: t(".start_button"),
-                           href: user_azure_ad_omniauth_authorize_path,
-                           as_button: true) %>
+    <% if Rails.configuration.x.mock_azure == "true" %>
```

Comment:
how tricky would it be to mack this config a. boolean so we an user something like 
```ruby 
if Rails.configuration.x.mock_azure?
...
```

Reply:
as discussed could do 
```ruby
...config.x.mock_azure = ENV.fetch("MOCK_AZURE")=="true" ? true : false
```